File: Pratica 6 - Funcoes 2/cpp_logic/exercicio9_v2.cpp
#include <iostream>
using namespace std;

int contar(string texto) {
    int cont = 0;
    for (char c : texto) {
        if (c=='a'||c=='e'||c=='i'||c=='o'||c=='u')
            cont++;
    }
    return cont;
}

int main() {
    string texto;
    cout << "Digite um texto: ";
    cin >> texto;

    cout << "Vogais: " << contar(texto);
    return 0;
}

File: Pratica 6 - Funcoes 2/java_logic/Exercicio9_v2.java
import java.util.Scanner;

public class Exercicio9_v2 {
    static int contar(String texto) {
        int cont = 0;
        for (char c : texto.toLowerCase().toCharArray()) {
            if ("aeiou".indexOf(c) != -1) cont++;
        }
        return cont;
    }

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.println("Digite uma frase ou uma palavra: ");
        String texto = sc.nextLine();
        System.out.println("Vogais: " + contar(texto));

        sc.close();
    }
}
